lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js

```diff
@@ -156,6 +156,76 @@ module.exports = [
     },
   },
   {
+    artifacts: {
+      TraceElements: [
+        {
+          'metricName': 'largest-contentful-paint',
+          'devtoolsNodePath': '0,HTML,1,BODY,1,DIV,0,IMG',
```

Comment:
We should leave out `devtoolsNodePath` in the smoke test to avoid the test becoming flaky. Also, I believe `nodeId` are run specific and I don't think we can guarantee their value every time.